Documentation/Getting Started with Open Disclosure.md

```diff
@@ -0,0 +1,85 @@
+
+* **Our Website**: https://www.opendisclosure.io
+* **GitHub Repositories**:
+  * **Front-End Repo**: https://github.com/caciviclab/odca-jekyll
+  * **Back-End Repo**: https://github.com/caciviclab/disclosure-backend-static
+
+## Communication Resources
+* **Slack**: Join the Open Oakland Slack at `openoakland.slack.com`, then join our specific `#open-disclosure` channel
+* **Waffle Board** (Kaban/Trello-style task-board): https://waffle.io/caciviclab/disclosure-backend
+
+## People To Know?
+Lots of people have worked on Open Disclosure in its 2+ years of existance (thanks for joining!), but if you *really* want to know "the deal" try these folks (@slack-Ids from openoakland.slack.com)
+
+* _Project Leads_: Tom Dooner(@tdooner) Aaron Borden(@adborden)
+* _Designer_: :warning::warning::warning:
+* _City Liason_: :warning::warning::warning:
+* ???: :warning::warning::warning:
+*
+*
+
+---
+
+# How to build the site on your machine
+
+Clone _*both*_ the back and front-end repos into the same parent directory. :warning: The front-end ***requires*** that the back-end directory be a sibling _specifically named_ `..\disclosure-backend-static` :warning: in order to build as it is hardcoded that way in the gulpfile.
+
+Dependencies you should already have installed: [Ruby **2.3.4**], [Bundler],
+[Make],
+[Node 8.9+],
+[ImageMagick] (Only need for production builds?? :warning: ?? **TODO** ?? :warning:),Python ?, [PostgreSQL] (only needed for back-end builds??)
+
+## ** Build the Front-End **
+(You did see the warning above that you'll need both repositories, right?)
+
+1. **1st-time only**: `cd` into the front-end directory, run `make setup`, then run `make build`.
+
+2. **Update** your local data resources with `make pull-finance`
+
+3. **Rebuild**/**Start Local Server** - *`make serve`*, then the site should be at http://localhost:4000/
+
+4. After making any changes you must run Front-End tests with `make test`.
+
+## ** Build the Back-End **
+If you are only doing front-end work, **building the entire back-end is not necessary** - you just need the base files to exist on your machine so feel free to skip this.
+
+* **1st-time only**: `cd disclosure-backend-static` and run the following three commands one-at-a-time:
+
+```
+sudo pip install -r requirements.txt
+gem install pg bundler
+bundle install
+```
```

Comment:
What do you think about just directing people to the `README` of the backend here, rather than copying the steps? Do you think it explains the setup well enough or could it be improved?